libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -455,7 +470,44 @@ class BreezSDKMapper {
             "message": messageSuccessActionData.message
         ]
     }
-    
+
+    static func dictionaryOf(receivePaymentRequest: ReceivePaymentRequest) -> [String: Any] {
+        return [
+            "amountSats": receivePaymentRequest.amountSats,
+            "description": receivePaymentRequest.description,
+            "preimage": arrayOf(preimageBytes: receivePaymentRequest.preimage),
+            "openingFeeParams": dictionaryOf(params: receivePaymentRequest.openingFeeParams)
+        ]
+    }
+
+    static func dictionaryOf(receivePaymentResponse: ReceivePaymentResponse) -> [String: Any] {
```

Comment:
openingFeeParams & openingFeeMsat are optional, so this should be `[String: Any?]`

Reply:
Done as part of df9aa9e